bedrock/firefox/templates/firefox/choose.html

```diff
@@ -0,0 +1,58 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-resp.html" %}
+
+{% block page_title_prefix %}{{_('Firefox Web Browser')}} - {% endblock %}
+{% block page_title %}{{_('Keep your identity yours with the new Firefox.')}}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/personal/logo.png') }}{% endblock %}
+{% block page_desc %}{{_('Choose Firefox when you want to browse the Web without the Web browsing you.')}}'{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_choose' %}
+{% endblock %}
+
+{% block tabzilla_tab %}{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block alt_header %}
+<div class="header-content">
+  <div id="tabzilla">
+    <a href="{{ url('mozorg.home') }}">Mozilla</a>
+  </div>
+  <h1>{{ high_res_img('firefox/choose/header-firefox.png', {'alt': _('Firefox'), 'width': '223', 'height': '84'}) }}</h1>
+
+  <h2 data-tracking="{{ _('Browse Freely.<br> Choose Firefox.') }}" data-selling="{{ _('Take Control.<br> Choose Firefox.') }}">{{_('Stand for choice.<br> Choose Firefox.')}}</h2>
+  {{ download_firefox(simple=true) }}
+</div>
+{% endblock alt_header %}
+
+{% block content %}
+<main role="main">
+  <div class="blurb">
+    <h2>{{ _('The most private browsing') }}</h2>
+    <p>{{ _('Take control of your web by using Private Browsing with Tracking Protection, only from Firefox.') }}</p>
+    <a href="{{ url('firefox.desktop.trust') }}" class="more">{{ _('Learn more about this') }}</a>
+  </div>
+
+  <div class="blurb">
+    <h2 class="data">{{ _('Here for you, not selling your info') }}</h2>
+    <p>{{ _('Discover easy tips about how to take back control of your online experience, only from Mozilla.') }}</p>
+    <a href="{{ url('firefox.desktop.trust') }}" class="more">{{ _('Get smart on data tracking') }}</a>
+  </div>
+</main>
+{% endblock %}
+
+{% block email_form %}{% endblock %}
+
+{% block site_footer %}
+  {% include 'firefox/includes/simple_footer.html' %}
+{% endblock %}
+
+{% block site_js %}
```

Comment:
Can just remove this block since it's not doing anything here in the inherited template?